jcl/src/jdk.management/share/classes/com/ibm/lang/management/MemoryMXBean.java

```diff
@@ -248,18 +248,18 @@
 	public String getGCMode();
 
 	/**
-     * Returns the amount of CPU time spent in the GC by the master thread, in milliseconds.
+     * Returns the amount of CPU time spent in the GC by the main thread, in milliseconds.
      * 
      * @return CPU time used in milliseconds
      */
-	public long getGCMasterThreadCpuUsed();
+	public long getGCMainThreadCpuUsed();
```

Comment:
The changes in this class are problematic, as they are publicly documented API. Changing these could break a user's application.
https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.api.80.doc/com.ibm.lang.management/com/ibm/lang/management/MemoryMXBean.html

Reply:
I think all we can do here is add the alternatives in addition to the original API, and deprecate the original for later removal.

Reply:
We'll also have to generate new docs.

The OpenJ9 version of the docs
https://www.eclipse.org/openj9/docs/api-langmgmt/
https://www.eclipse.org/openj9/docs/api/jdk8/jre/management/extension/com/ibm/lang/management/MemoryMXBean.html

Reply:
I have added back and deprecated the master/slave public methods and created a docs issue: https://github.com/eclipse/openj9-docs/issues/607